PubnativeLite/PubnativeLite/AppMonet/AppMonet.m

```diff
@@ -0,0 +1,47 @@
+////
+//  Copyright © 2020 PubNative. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import "AppMonet.h"
+#import "HyBid.h"
+
+static BOOL testMode = NO;
```

Comment:
Should it be static? It is global anyway, no?

Reply:
The problem or the question that I have in my mind is how to disable testMode after we are done with it? I couldnt see any mehtods for it. I've asked the same question to @eros902002 since they have the same thing for Android.

Reply:
Here same as in Android. I wouldn't toggle the test mode, I'd just keep it as they do in their interface and if it is a limitation then publishers are already aware of it. I'd rather go with that than having the SDK behave in a way they don't expect.

Reply:
Okay then, I'll remove the toggle and call `[HyBid setTestMode:YES];` directly here @orkhanalizada2020 @eros902002  👍